bin/oref0-pump-loop.sh

```diff
@@ -423,7 +423,9 @@ function smb_bolus {
     && if (grep -q '"units":' enact/smb-suggested.json 2>&3); then
         # press ESC four times on the pump to exit Bolus Wizard before SMBing, to help prevent A52 errors
         echo -n "Sending ESC ESC ESC ESC to exit any open menus before SMBing "
-        try_return mdt -f internal button esc esc esc esc 2>&3 \
+        try_return mdt -f internal button esc 2>&3 \
```

Comment:
Let’s do two here. Having an even total number of ESCs puts the pump back on the home screen, vs the info screen you get with one ESC.

Reply:
done.